packages/trpc/src/procedures/recipes/deleteRecipe.spec.ts

```diff
@@ -8,6 +8,7 @@ import { faker } from "@faker-js/faker";
 describe("deleteRecipe", () => {
   let user: User;
   let trpc: CreateTRPCProxyClient<AppRouter>;
+  jest.setTimeout(50000);
```

Comment:
There's a global jest configuration flag that I believe you can pass in via the CLI to extend test timeouts. Unfortunately we can't commit these to the repository.